packages/tldraw/src/lib/utils/buildFromV1Document.ts

```diff
@@ -553,8 +553,8 @@ export function buildFromV1Document(editor: Editor, document: LegacyTldrawDocume
 								})
 
 								const handles = editor.getShapeHandles(v2ShapeFresh)!
-
-								const change = util.onHandleChange!(v2ShapeFresh, {
+								const initial = undefined
```

Comment:
I wasn't too sure what this file was, this seemed the best way to deal with not having access to the initial shape. 🤷🏽‍♂️

Reply:
should be fine. This file is for loading old tldraw v1 file data into the new v2 format.